.circleci/Makefile

```
@@ -1,4 +1,6 @@
-all default: config.yml
+.PHONY: all test clean default
+default: all
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)